### DISCUSSION TITLE
Shell command example

### DISCUSSION THREAD

**Discussion by 3goats** [tauri-apps/tauri#10163] [Q&A]
Hi I just bootstrapped a demo Svelte app using Tauri. I followed the tutorial here - https://tauri.app/v1/guides/getting-started/setup/sveltekit/. Whilst this provides a basic great example, they don't show how to use some of the other OS integration utilities such as "command". Is there an example of how to do this anywhere using something simple like `ls` ?

**Comment by bcahlit** [comment]
**https://tauri.app/v1/api/js/shell/#__docusaurus_skipToContent_fallback**

**https://v2.tauri.app/plugin/shell/**

i have same problem， These two links may help
